example/lib/main.dart

```diff
@@ -70,8 +71,6 @@ class _TorchAppState extends State<TorchApp> {
   }
 
   _showMessage(String message) {
-    _scaffoldKey.currentState.showSnackBar(SnackBar(
-      content: Text(message),
-    ));
+    _messangerKey.currentState?.showSnackBar(SnackBar(content: Text(message)));
```

Comment:
Please use `ScaffoldMessenger.of(context)`, there's no need to maintain a var with this key, so the var `_messangerKey` can be removed. Thanks.

Reply:
Because of [this issue I encountered](https://stackoverflow.com/questions/66833689/flutter-no-scaffoldmessenger-widget-found), I moved the whole Scaffold into a stateless widget that I named "TorchController" as you can see in my latest commit :) 

Everything seems to work perfectly 👌